src/api/entities/confidential/ConfidentialAsset/types.ts

```diff
@@ -16,3 +16,26 @@ export interface GroupedAuditors {
   auditors: ConfidentialAccount[];
   mediators: Identity[];
 }
+
+export interface CreateConfidentialAssetParams {
+  /**
+   * Custom data to be associated with the confidential Asset
```

Comment:
```suggestion
   * Custom data to be associated with the Confidential Asset
```

Confidential feels inseparable from Asset to me. Plus its consistent with "Confidential Account" casing